lib/library/library.ts

```diff
@@ -0,0 +1,489 @@
+    'Tar’tari Tendary: the City of Cosmic Shadows',
+    'Secrets and Riddles of the Weave: Vol 14',
+    'Shadow and Dust: What Rots and Remains',
+    'Fey Unbound: Study of Entrapped Creatures and their Release',
+    'Evil’s Horde: Hugar and Juik’s Notes On Demons',
+    'Asamokology: the Nature of What Is Fake',
+    'Breaking the Cycle: the Written War Against the Gods - Vol. 146',
+    'Losing Yourself To Be One: Mind Flayer and the Hive Mind',
+    'Stone Eyes: the Myth of Aqytorky',
+    'The Fire Within the Stone: A Divination Primer for Beginners',
+    'You Can Call Me Gary: The Autobiography of Lord Gygax',
+    'Two Runes and a Pentagram: Summoning Demons for Beginners',
+    "The King's Consort - Vol. 8",
+    'Back to Work Peasants! The Writings of The Bad King Quellix'
+  ],
+  puns: [
+    'How to make Illusions and Charm People',
+    'The Illithid by Homer',
+    'Memoirs of a Genasi',
+    "The 7 Habits of Highly Effective Peasants (The sequel to 'The Hunger Games: a True Story')",
+    'Diary of a Wimpy Kobold',
+    "A Handmaid's Tail: a collection of short stories by notable Tieflings",
+    'Ready Mindflayer One',
+    'Dragon Turtles All the Way Down',
+    'The Lion, the Witch, and the War-forged',
+    'Gone with the Healing Wind',
+    'To Kill a Manticore',
+    'Planar-shifting for People in a Hurry',
+    "One Flew Over the Kenku's Nest",
+    'The Brothers Dragonbornov',
+    'For Whom the Behir Tolls',
+    'The Power of Mimics: Why Certain Encounters Have Extraordinary Impact',
+    'The Wizard of Ooze',
+    'Fight. Club. - A Comprehensive Guide for Barbarians',
+    'The Secret Life of Bards',
+    'The Giving Treant',
+    "Alice's Adventures in the Underdark",
+    '50 Shades of Fey',
+    'Sense Motive and Sensability',
+    'As I Lay at Zero Hit Points',
+    'Great Incantations',
+    'The Amityville Hook Horror',
+    "Dante's Infernal",
+    'The Girl with the Dragonborn Tattoo',
+    "Lolth's Web",
+    'Satyrs and Sensibility',
+    'Pride and Prestidigitation',
+    'The Constant Scrivener',
+    'The Left Hand Casts Darkness',
+    'The Cockatrice in the Rye',
+    'One Thousand and One Knights',
+    'The Grapes of Wraith: Wining and Dining the Undead',
+    'The Adventures of Nancy Druid',
+    '1984 Orcs',
+    'Of Kobolds and Men',
+    'The Great Goliath',
+    'Animals Farm, Gardening Tips From a Friendly Druid',
+    "Little Women, A Female Halfling in a Human's World",
+    'Lord of the Gnolls',
+    'The Orc Man and the Sea',
+    'Moby-Dick, or, the Kraken',
+    "Hairy Kobold and the Sorcerer's Throne",
+    'Hairy Kobold and the Chamber of Regrets',
+    'Hairy Kobold and the Prisoner of Gundabad',
+    'Hairy Kobold and the Hobbit of Fire',
+    'Hairy Kobold and the Larder of the Phoenix',
+    'Hairy Kobold and the Half-Elf Prince',
+    'Hairy Kobold and the Deathly Horsemen',
+    'The Giving Treent',
+    'The Call of the Wildshape'
+  ]
+}
+
+export const library = {
```

Comment:
Empty object can be removed?

Reply:
Not yet- the PR isn't even close to being complete, it's just so people can see what's being worked on so we don't get duplicates.